STYLEGUIDE.md

```diff
@@ -1,165 +0,0 @@
-# Style Guide
```

Comment:
Can we symlink this, or just keep an empty doc with a link to the site? Because I think GitHub encourages repos to contain this file in the root.

Reply:
I checked at 
- https://medium.com/code-factory-berlin/github-repository-structure-best-practices-248e6effc405 
- https://github.com/joelparkerhenderson/github-special-files-and-paths

and neither lists `STYLEGUIDE` as a "standard" file of which GitHub is in any way aware.

Reply:
I mixed it up with the `CONTRIBUTING.md` doc.